samples/pipeline-tests.sh

```diff
@@ -1,110 +1,196 @@
 
-seldon model load -f ./models/mul10.yaml
-seldon model load -f ./models/add10.yaml
+load model ./models/mul10.yaml
+load model ./models/add10.yaml
 seldon model status mul10 -w ModelAvailable | jq -M .
 seldon model status add10 -w ModelAvailable | jq -M .
-seldon pipeline load -f ./pipelines/trigger-joins.yaml
+load pipeline ./pipelines/trigger-joins.yaml
 seldon pipeline status trigger-joins -w PipelineReady | jq -M .
 seldon pipeline infer trigger-joins --inference-mode grpc  '{"model_name":"pipeline","inputs":[{"name":"ok1","contents":{"fp32_contents":[1]},"datatype":"FP32","shape":[1]},{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
 seldon pipeline infer trigger-joins --inference-mode grpc  '{"model_name":"pipeline","inputs":[{"name":"ok3","contents":{"fp32_contents":[1]},"datatype":"FP32","shape":[1]},{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
-seldon pipeline unload trigger-joins
-seldon model unload mul10
-seldon model unload add10
+unload pipeline trigger-joins ./pipelines/trigger-joins.yaml
+unload model mul10 ./models/mul10.yaml
+unload model add10 ./models/add10.yaml
+
+
+# MLServer
+sleep $sleepTime
+load model ./models/sklearn-iris-gs.yaml
+seldon model status iris -w ModelAvailable | jq -M .
+seldon model infer iris '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}' 
+seldon model infer iris --inference-mode grpc \
+   '{"model_name":"iris","inputs":[{"name":"input","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[1,4]}]}' | jq -M .
+unload model iris ./models/sklearn-iris-gs.yaml
+
+
+# Experiments
+load model ./models/sklearn1.yaml
+load model ./models/sklearn2.yaml
+seldon model status iris -w ModelAvailable
+seldon model status iris2 -w ModelAvailable
+seldon model infer iris -i 50 \
+  '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}'
+seldon model infer iris2 -i 50 \
+  '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}'
+load experiment ./experiments/ab-default-model.yaml 
+seldon experiment status experiment-sample -w | jq -M .
+seldon model infer iris -i 50 \
+  '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}'
+seldon model infer iris --show-headers \
+  '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}'
+seldon model infer iris -s -i 50 \
+  '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}' 
+seldon model infer iris --inference-mode grpc -s -i 50\
+   '{"model_name":"iris","inputs":[{"name":"input","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[1,4]}]}'
+unload experiment experiment-sample ./experiments/ab-default-model.yaml
+unload model iris ./models/sklearn1.yaml
+unload model iris2 ./models/sklearn2.yaml
```

Comment:
nit; github complains that there's no newline at end of file

Reply:
done